api/sql/schema/750$transfer.push.confirmReversal.sql

```diff
@@ -23,6 +23,14 @@ BEGIN TRY
     WHERE
         secondTransferId=@transferId
 
+    EXEC [transfer].[push.event]
+        @transferId = @transferId,
+        @type = 'transfer.reverse',
+        @state = 'reversed',
```

Comment:
use 'reverse' as in other places we use words like 'confirm', 'fail'
also have a look at this fix: https://github.com/softwaregroup-bg/ut-transfer/commit/ac78f86c0438816f850047126549a960697cb16c